src/python/pants/init/engine_initializer.py

```diff
@@ -242,13 +235,13 @@ class GoalMappingError(Exception):
   def _make_goal_map_from_rules(rules):
     goal_map = {}
     goal_to_rule = [(rule.goal, rule) for rule in rules if getattr(rule, 'goal', None) is not None]
-    for goal, rule in goal_to_rule:
+    for goal, r in goal_to_rule:
```

Comment:
This rename doesn't seem like much of a win. I often like short names, but only when it's for highly generic code / functional programming.

Reply:
After importing `rule`, the linter was complaining that the loop variable was shadowing the import.

Reply:
Ah that's fair. Maybe for parity make it be `for g, r`? Not sure if that's worse or better.

Reply:
That's annoying. I like `goal, r`, as the `r` being rule can be deduced from the variable `goal_to_rule`'s name.